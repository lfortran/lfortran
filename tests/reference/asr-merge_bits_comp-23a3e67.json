{
    "basename": "asr-merge_bits_comp-23a3e67",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/merge_bits_comp.f90",
    "infile_hash": "07da68f4c47a0ded636941ace875a8d1c0c31abec944e32405b95162",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-merge_bits_comp-23a3e67.stderr",
    "stderr_hash": "d1fab14201f28c9009c33c460571505646f71a4151917b273ffde9e3",
    "returncode": 2
}