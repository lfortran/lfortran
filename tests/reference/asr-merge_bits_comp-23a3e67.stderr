semantic error: Kind of all the arguments of Mergebits must be the same
 --> tests/errors/merge_bits_comp.f90:2:14
  |
2 |     print *, merge_bits(8, 12_8, 2)
  |              ^^^^^^^^^^^^^^^^^^^^^^ 
